src/views/Nft/market/types/MarketEvent.tsx

```diff
@@ -0,0 +1,7 @@
+export enum MarketEvent {
```

Comment:
Let's move this and Activity.tsx to `src/views/Nft/market/types.ts`

Reply:
I couldn't move there it gave me dependency cycle error. I moved to `state/nftMarket/types.ts` instead